src/errors/test-run/templates.js

```diff
@@ -23,6 +23,27 @@ function markup (err, msgMarkup) {
     return msgMarkup;
 }
 
+function formatOrdinalNumber (number) {
+    if (number < 4 || number > 20) {
+        switch (number % 10) {
+        case 1: return number + 'st';
+        case 2: return number + 'nd';
+        case 3: return number + 'rd';
+        }
+    }
+
+    return number + 'th';
+}
+
+function formatArrayArgumentErrorReason (err) {
+    if (err.invalidLength)
+        return `its <code>length</code> property was <code>${err.actualValue}</code>`;
+    else if (err.elementIndex !== void 0)
+        return `its ${formatOrdinalNumber(err.elementIndex)} element was <code>${err.actualValue}</code>`;
```

Comment:
Can we just use phrase `element at index {num}` to avoid dealing with ordinal number formatting?